source/usage-and-configuration/swagger/swagger-to-html.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Swagger to HTML
+---
+
+# Swagger to HTML
```

Comment:
Put (beta) in the title, remove ** *Note: This feature is in beta phase.* **

Reply:
👍